packages/now-cli/src/commands/env/pull.ts

```diff
@@ -0,0 +1,99 @@
+import chalk from 'chalk';
+import { NowContext, ProjectEnvTarget } from '../../types';
+import { Output } from '../../util/output';
+import Client from '../../util/client';
+import stamp from '../../util/output/stamp';
+import getEnvVariables from '../../util/env/get-env-records';
+import getDecryptedSecret from '../../util/env/get-decrypted-secret';
+import { getLinkedProject } from '../../util/projects/link';
+import cmd from '../../util/output/cmd';
+import withSpinner from '../../util/with-spinner';
+import { join } from 'path';
+import { promises } from 'fs';
+import { emoji, prependEmoji } from '../../util/emoji';
+const { writeFile } = promises;
```

Comment:
Wow 😍

Reply:
Yep, [`fs.promises`](https://nodejs.org/api/fs.html#fs_fspromises_writefile_file_data_options) was added in Node 10 so we can use it now that Now CLI dropped Node 8 support in January 😄